README.md

```diff
@@ -25,16 +25,16 @@ $ cargo install star-history
 ## Usage
 
 We require a token for accessing GitHub's GraphQL API. Head to
-https://github.com/settings/tokens and click "Generate new token". The default
-public access permission is sufficient &mdash; you can leave all the checkboxes
-empty. Save the generated token somewhere like ~/.githubtoken.
+https://github.com/settings/tokens and click "Generate new token". Organizations require
+the `read:org` permission. Save the generated token somewhere like ~/.githubtoken.
```

Comment:
I tested this PR with a token that has only "public access" permission (no `read:org`) and it seems to work. Is the permission only relevant in the case that you want to count stars of a private repo?

Reply:
I got an error while trying `query.organization`, but I didn't test whether it was still required for `query.repositoryOwner`. 
It works without `read:org` now, thanks for pointing that out!